### PR TITLE
Add Luminr.co in JSON fromat

### PR DESCRIPTION
**Pull request by Br9ha20**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/luminr-co.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "socials": {
+      "facebook": "https://www.facebook.com/profile.php?id=100093330643165",
+      "github": "https://github.com/luminr-co/luminr.co",
+      "instagram": "https://www.instagram.com/luminr.co/",
+      "linkedin": "https://www.linkedin.com/company/luminr-co/",
+      "twitter": null,
+      "youtube": null
+    }
+  }
+
```
Comment (thenishantsapkota) [inline data/luminr-co.json:17]: Could you please add a new line at the end of the file.

File: data/luminr-co.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Luminr",
+    "establishedAt": 1697548800000,
+    "location": "Kaldhara, Kathmandu",
+    "website": "https://luminr.co//",
+    "email": "business@luminr.co",
+    "phone": "+977-9860050391",
```
Comment (thenishantsapkota) [inline data/luminr-co.json:7]: If this place has a landline number, please add it here.

File: data/luminr-co.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Luminr",
+    "establishedAt": 1697548800000,
```
Comment (thenishantsapkota) [inline data/luminr-co.json:3]: This timestamp is for `October 17th 2023`. The field is the company established date

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/luminr-co.json (hunk 2 above)
Comment (Br9ha20) [reply data/luminr-co.json:7]: There is no landline right now, unfortunately, when it's added I will update it accordingly.

File: data/luminr-co.json (hunk 3 above)
Comment (Br9ha20) [reply data/luminr-co.json]: Changed to the correct date

File: data/luminr-co.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Luminr",
```
Comment (thenishantsapkota) [inline data/luminr-co.json:2]:
```suggestion
  "name": "Luminr Co",
```

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for the contribution @Br9ha20 <3

**Pull request merged**